docs/configure-archive-logs.md

```diff
@@ -59,3 +61,23 @@ spec:
     archiveLocation:
       archiveLogs: true
 ```
+
+## Suggested alternatives
+
+Argo Workflows log storage facilities are quite basic.
+
+We recommend using a fully featured Kubernetes-aware logging facility which will provide you with indexing, searching and managing of log storage.
+
+We do not plan to reach feature parity with purpose built-logging facilities as there are more optimized tools available in the ecosystem. Some open-source tools which are known to work:
```

Comment:
With these 3 paragraphs right adjacent to each other now, I hope it's clear that they are quite repetitive of each other; they can be consolidated into one.

```suggestion
Argo's log storage is naive and will not reach feature parity with purpose-built facilities optimized for indexing, searching, and storing logs. Some open-source tools include:
```
- also fixes typo "purpose built-logging" -> "purpose-built"
- same as previous comment, "which are known to work" can be made simpler and more direct
- same as previous comments on "Argo Workflows" -> "Argo"